Instruments/source/CacheTupleProducer.cxx

```diff
@@ -165,6 +173,7 @@ class CacheTupleProducer {
     std::vector<SignalObjectSelector> signalObjectSelectors;
     std::vector<UncertaintySource> unc_sources;
     std::vector<JetOrdering> vector_jet_ordering;
+    std::shared_ptr<analysis::tools::ProgressReporter> progressReporter;
```

Comment:
don't need to be a shared_ptr, `tools::ProgressReporter progressReporter;` should do.